great_expectations/data_context/data_context.py

```diff
@@ -12,13 +12,14 @@
 import warnings
 import webbrowser
 from collections import OrderedDict
-from typing import Any, Callable, Dict, List, Optional, Union, cast
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union, cast
 
 import requests
 from dateutil.parser import parse
 from ruamel.yaml import YAML, YAMLError
 from ruamel.yaml.comments import CommentedMap
 from ruamel.yaml.constructor import DuplicateKeyError
+from typing_extensions import Literal
```

Comment:
A 3.8 concept so we need to use `typing_extension`. This gives us access to a lot of really useful tools.